subsys/net/lib/sockets/Kconfig

```diff
@@ -62,6 +62,12 @@ config NET_SOCKETS_TLS_MAX_CIPHERSUITES
 	  By default, all ciphersuites that are available in the system are
 	  available to the socket.
 
+config NET_SOCKETS_OFFLOAD
+	bool "Offload Socket APIs"
+	select NET_SOCKETS_POSIX_NAMES
+	help
+	Enables direct offloading of socket operations to dedicated TOE hardware
```

Comment:
Also add information about restrictions when this offloading should not be used (multiple network interface with routing etc).
BTW, what does "TOE" here means, please open this up.

Reply:
Yes, OK.
TOE is a general term, which I think applies to this case:  
https://en.wikipedia.org/wiki/TCP_offload_engine